safe-global/safe-wallet-web
# fix: Remove SAP banner

## What it solves

Resolves [SW-378](https://www.notion.so/safe-global/Remove-Safe-Pass-Banner-1288180fe57380e5a6cbc1ad2a235f54)

## How this PR fixes it

- Removes the SAP banner from the dashboard

## How to test it

1. Open the dashboard in a new browser
2. Observe no SAP Banner

## Screenshots

## Checklist
